src/styles/card.scss

```diff
@@ -0,0 +1,39 @@
+@import 'variables';
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  border-radius: 5px;
+  border: 1px solid $color-border-card;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    background-size: contain;
+    background-repeat: no-repeat;
+    margin: 32px 19px 36px;
+  }
+
+  &__title {
+    font-size: 12px;
+    line-height: 18px;
+    width: 166px;
+    height: 36px;
+    margin: 0 16px 4px;
+    font-family: $font-roboto;
+    font-weight: $font-medium;
+    color: $color-font;
+  }
+
+  &__subtitle {
+    font-size: 10px;
+    line-height: 14px;
+    margin: 0 16px 16px;
+    font-family: $font-roboto;
+    font-weight: $font-regular;
+    height: 14px;
+    width: 98px;
+    color: $color-font-secondary;
+  }
```

Comment:
Similarly, for `.card__subtitle`, using fixed width and height can be problematic for the reasons mentioned above. Ensure that text does not overflow or get clipped by allowing the container to adjust to the content.